### PR TITLE
Add seed

### PR DESCRIPTION
**Pull request by tinglycraniumplacidly**

### CODE REVIEW

**Comment by aattsai** [comment]
:+1:  :+1:

File: db/seeds.rb [hunk 1]
```diff
@@ -351,3 +353,492 @@
     end
   end
 end
+
+###############################################################################
+claudio = Restaurant.create(name: "Da Claudio Ristorante",
+                                      website: "http://www.daclaudionyc.com",
```
Comment (dijonkitchen) [inline db/seeds.rb]: Check indentation.

File: db/seeds.rb [hunk 2]
```diff
@@ -351,3 +353,492 @@
...
+  description: "With parmigiano, olio di oliva, limone",
+  featured: false)
+
+carpaccio.taste_list = "salty"
+carpaccio.ingredient_list = "olives, lemon"
+
+u1.favorites.create(item_id: carpaccio.id)
+u2.favorites.create(item_id: carpaccio.id)
+u3.favorites.create(item_id: carpaccio.id)
+carpaccio.save
```
Comment (dijonkitchen) [inline db/seeds.rb]: Move this up a few lines.

File: db/seeds.rb [hunk 3]
```diff
@@ -351,3 +353,492 @@
...
+  price: "14.00",
+  photo_url: "",
+  description: "With spicy sopressata, balsamiv, pecorino cheese, caramelized walnuts",
+  featured: false)
+
+insalata.taste_list = "spicy, salty"
+insalata.ingredient_list = "cheese, walnuts"
+insalata.diet_list = ""
+
+u1.favorites.create(item_id: insalata.id)
```
Comment (dijonkitchen) [inline db/seeds.rb:487]: Make favorites placement consistent below tagging.

**Pull request merged**